chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -309,6 +312,10 @@ def cinder_volume
           @node[:block_device][@device][:vendor] == "cinder" && @node[:block_device][@device][:model] =~ /^volume-/
         end
 
+        def mtu
+          @mtu || 1500
+        end
```

Comment:
We don't need this method now, do we? With `attr_reader`, calling `mtu` will just fetch `@mtu`.